app/Traits/AccessControlAPI.php

```diff
@@ -124,6 +124,7 @@ private function bulkAccessControl($api_key, $fileset_hash)
                 ->where(function ($query) use ($continent) {
                     $query->where('continent_id', $continent);
                 })
+                ->select('id', 'name')
```

Comment:
We only need to select the id and name column because in the line 130 will only validate if a record exists.